js/components/controlBar.js

```diff
@@ -879,33 +876,38 @@ var ControlBar = createReactClass({
         alignment = CONSTANTS.TOOLTIP_ALIGNMENT.LEFT;
       }
       tooltipAlignments[collapsedControlBarItems[k].name] = alignment;
-      finalControlBarItems.push(controlItemTemplates[collapsedControlBarItems[k].name]);
+      var item = controlItemTemplates[collapsedControlBarItems[k].name];
+      if (this.props.equalSpacing) {
+        item = (
+          <div className="oo-flex-row" key={collapsedControlBarItems[k].name}>
```

Comment:
Is it necessary to add an extra container here? I think this might make it slightly more complex to maintain/update the CSS and we also have two places where the key is set. Could we do something like:

- Add a modifier class at the control bar level and use it customize the styles of `oo-control-bar-item` elements 
- Add an additional class to the control bar buttons when this mode is active